openhcl/hcl/src/ioctl/aarch64.rs

```diff
@@ -96,6 +101,9 @@ impl super::BackingPrivate for MshvArm64 {
                 true
             }
             HvArm64RegisterName::X18 => {
+                if vtl == GuestVtl::Vtl1 {
+                    todo!("TODO: handle X18 for VTL1");
+                }
```

Comment:
I think this could hit, e.g. in the hypercall handling path. Maybe let's keep it as a normal TODO comment for now.

Reply:
Yeah, good idea!